projects/file-parsing/types-converting/types.go

```diff
@@ -0,0 +1,16 @@
+package filesConverting
+
+type Score struct {
+	Name      string
+	HighScore int32 `json:"high_score"`
+}
+
+func Want() []Score {
```

Comment:
It's unusual that this is in your production code package, rather than a test file. This is test-specific data, so I'd expect it to be in a test file, or a package that only exists for testing data/utilities.